packages/adyen-integration/src/adyenv3/adyenv3.ts

```diff
@@ -881,3 +881,9 @@ export function isCardState(param: unknown): param is CardState {
         typeof (param as CardState).data.paymentMethod.encryptedExpiryMonth === 'string'
     );
 }
+
+export interface AdyenPaymentMethodInitializationData {
+    clientKey?: string;
```

Comment:
All fields look required. Am I missing something?

Reply:
I get this types as the same as it used for [AdyenConfiguration](https://github.com/bigcommerce/checkout-sdk-js/blob/43b963d4f04ad461b3721b8484c8a1dbcb657f26/packages/adyen-integration/src/adyenv3/adyenv3.ts#L211)
If they should be required I will update them?